odinmain/src/main/kotlin/me/odinmain/commands/impl/DevCommand.kt

```diff
@@ -108,7 +108,7 @@ val devCommand = commodore("oddev") {
         val xPos = -185 + x * 32
         val zPos = -185 + z * 32
         val core = ScanUtils.getCore(xPos, zPos)
-        val northPos = DungeonUtils.Vec2(xPos, zPos - 4)
+        val northPos = Vec2(xPos, zPos - 4)
```

Comment:
Just remove this from the message, it's redundant as the scanning no longer uses rotation cores.